### PR TITLE
Branch 2

### PR DESCRIPTION
**Pull request by jakeChal**

### CODE REVIEW

File: bla.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Me lene Giorgo
```
Comment (nefeliKa) [inline bla.md:1]: Me lene Taki

File: bla2.md [hunk 2]
```diff
@@ -0,0 +1 @@
+foobar
```
Comment (nefeliKa) [inline bla2.md:1]: bla

Comment (jakeChal) [reply bla2.md:1]: no this is correct

File: bla.md (hunk 1 above)
Comment (jakeChal) [reply bla.md]: reworked

**Pull request merged**